### ISSUE TITLE
[Request] linux version of chrome?

### ISSUE THREAD

**Issue by supechicken666** [skycocker/chromebrew#4298]
My chromebook is end-of-life already and don't have update anymore
Due to my school, I need to use newest version of chrome to access the school intranet server
I can't install crouton as the storage space doesn't enough
And I only can use chrome to access it 
Thanks

**Comment by uberhacker** [comment]
@supechicken666: See https://www.ifixit.com/News/30282/how-to-get-updates-on-your-end-of-life-chromebook.

**Comment by ThatGeekyWeeb** [comment]
There isn't "great" support here for CloudReady, so you could use Chomefy, but that requires a bit of extra work, but *should* work on all arches.
> https://github.com/imperador/chromefy

**Comment by ThatGeekyWeeb** [comment]
Additionally I'll look into packaging Chrome.

**Comment by supechicken666** [comment]
Thanks for suggest
Does the cloudready support my exynos chromebook?

**Comment by ThatGeekyWeeb** [comment]
As long as your Chromebook is x86 based CloudReady should work.

**Comment by uberhacker** [comment]
https://guide.neverware.com/supported-devices/

**Comment by supechicken666** [comment]
I have 3 question in mind:
I found that both the cloudready and the mr chromebox supported list don't contain my chromebook, can i still continue install?
Just checked my chromebook specs with the list on wikipedia, It said my chromebook is aarch64 arch. Have cloudready support it?
How to install chromefy on arm devices? (the readme didnt list)

**Comment by uberhacker** [comment]
If your device is not listed, YMMV.  See https://cloudreadykb.neverware.com/s/article/Installing-CloudReady-On-An-Uncertified-Device.

**Comment by supechicken666** [comment]
It didn’t show the step of chromebook? What I need to do after creating the recovery usb?

**Comment by ThatGeekyWeeb** [comment]
The readme for Chromefy isn't "great", but the process is the same for all arches.
1. Take a look at [this list](https://www.chromium.org/chromium-os/developer-information-for-chrome-os-devices), and find your device, find another device with the same arch, remember it's codename.
2. Follow [`option-1-automated-scrip`](https://github.com/imperador/chromefy#option-1-automated-script---generating-your-own-chrome-img), but remove the usage of `swtpm.tar` or `Caroline`'s recovery image.

Note: That Choefy isn't meant for Chromebooks, and the script requires some edits, I'll post my edited script here soon:tm:

**Comment by ThatGeekyWeeb** [comment]
And, after looking into it, since your device is arm64 based, you'll have to use Chromium, although, this will leave you without the ability to use Flash or `libwidevine` encryption-based content.
***
Additionally, Void Linux doesn't build Chromium for `arm*` so idk if I can package it, since my device is armv7, and since your using arm64, you might also be out of luck.
> `only_for_archs="i686 x86_64 x86_64-musl"`

**Comment by supechicken666** [comment]
I guess the armv7 app can install and run in armv8 or arm64 like android
Also I will try on Chromefy tomorrow

**Comment by supechicken666** [comment]
I found my device is arm arch (is that mean armv7? Not aarch64?) on this site 
https://www.chromium.org/chromium-os/developer-information-for-chrome-os-devices
But another website tells me mine is arm64?
By the way I installed the chromebook duet armv7 modify image by script you give me an hour ago
It can boot and open browser, but the screen will frozen and turn black after using a while.
Is it a bug?
